Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_time.py

```diff
@@ -301,36 +302,36 @@ def match_to_time(self, match: Match, reference: datetime):
             min_str = RegExpUtility.get_group(match, 'min')
```

Comment:
Here we have to replace the 'min' string by its constant: min_str = RegExpUtility.get_group(match, Constants.MIN_NUM)

Reply:
Same on line 339. It should be: time_prefix = RegExpUtility.get_group(match, Constants.PREFIX_GROUP_NAME)